virtual_tx_builder.go

```diff
@@ -0,0 +1,22 @@
+package taprootassets
+
+import (
+	"github.com/btcsuite/btcd/wire"
+	"github.com/lightninglabs/taproot-assets/asset"
+	"github.com/lightninglabs/taproot-assets/tapscript"
+)
+
+// GroupTxBuilder is an implementation of the asset.GenesisTxBuilder interface
+// that constructs virtual transactions for grouped asset genesis.
+type GroupTxBuilder struct{}
+
+func (b *GroupTxBuilder) BuildGenesisTx(newAsset *asset.Asset) (*wire.MsgTx,
```

Comment:
Looks like the 3rd instance of this in the diff? Wondering if we can consolidate somewhat. Might need to modify the structure of the mock packages and/or tests.